.github/workflows/release-images.yaml

```diff
@@ -0,0 +1,338 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Build Images
+jobs:
+  builde2e:
+    strategy:
+      fail-fast: false
+      max-parallel: 10
+      matrix:
+        os:
+        - "centos 7.9"
+        - "redhat 7.9"
+        - "redhat 8.4"
+        - "redhat 8.6"
+        - "oracle 7.9"
+        - "rocky 9.0"
+        - "rocky 9.1"
+        - "sles 15"
+        - "ubuntu 20.04"
+        buildConfig:
+        - "basic"
+        - "fips"
+        - "nvidia"
+        - "offline"
+        - "offline-fips"
+        - "offline-nvidia"
+        cloud_provider:
+        - "aws"
+        - "azure"
+        - "gcp"
+        - "ova"
+        exclude:
+        - os: "centos 7.9"
```

Comment:
this exclude list is crazy. 🧠 
let me try to check if `include` works better.

Reply:
moved to a new PR